relentless/simulate/simulate.py

```diff
@@ -21,6 +24,12 @@
     Potentials
     PotentialTabulator
     PairPotentialTabulator
+    Thermostat
```

Comment:
I think `Thermostat` and `Barostat` should go in the developer section, since they are essentially abstract bases.